os/board/rtl8721csm/src/component/common/bluetooth/realtek/sdk/example/ble_scatternet/ble_tizenrt_scatternet_app.c

```diff
@@ -688,6 +673,21 @@ void ble_tizenrt_scatternet_app_handle_conn_state_evt(uint8_t conn_id, T_GAP_CON
                 } else {
                     debug_print("\n[%s] Memory allocation failed", __FUNCTION__);
                 }
+            } else {
+                if(GAP_CAUSE_SUCCESS == le_bond_delete_by_bd(ble_tizenrt_scatternet_app_link_table[conn_id].remote_bd,
+                                ble_tizenrt_scatternet_app_link_table[conn_id].remote_bd_type))
+                {
+                    debug_print("\r\n[%s] Delete bond success", __FUNCTION__);
+                } else {
+                    debug_print("\r\n[%s] Delete bond fail!!!", __FUNCTION__);
+                }
+                memset(&ble_tizenrt_scatternet_app_link_table[conn_id], 0, sizeof(BLE_TIZENRT_SCATTERNET_APP_LINK));
+                uint32_t connid = (uint32_t) conn_id;
+                if(ble_tizenrt_scatternet_send_callback_msg(BLE_TIZENRT_DISCONNECTED_MSG, (void *) connid) == false)
+                {
+                    debug_print("\r\n[%s] callback msg send fail", __FUNCTION__);
+                    printf("\r\n[%s] callback msg send fail", __FUNCTION__);
```

Comment:
Why do you use both `debug_print` and `printf`? How about merging?

Reply:
Fixed.